packages/zendesk-adapter/src/filters/audit_logs.ts

```diff
@@ -0,0 +1,315 @@
+import { collections } from '@salto-io/lowerdash'
+import { FilterCreator } from '../filter'
+import { AUDIT_TIME_TYPE_NAME, TRANSLATION_TYPE_NAMES, ZENDESK } from '../constants'
+import ZendeskClient from '../client/client'
+import { getIdByName } from '../user_utils'
+import { FETCH_CONFIG, GUIDE_GLOBAL_TYPES, GUIDE_TYPES_TO_HANDLE_BY_BRAND } from '../config'
+
+const log = logger(module)
+const { awu } = collections.asynciterable
+
+export const AUDIT_TIME_TYPE_ID = new ElemID(ZENDESK, AUDIT_TIME_TYPE_NAME)
+const AUDIT_TIME_INSTANCE_ID = AUDIT_TIME_TYPE_ID.createNestedID('instance', ElemID.CONFIG_NAME)
+const ELEMENTS_WITH_PARENTS = [
+  'ticket_field__custom_field_options',
+  'user_field__custom_field_options',
+  'organization_field__custom_field_options',
+  'business_hours_schedule_holiday',
+]
+const GUIDE_ELEMENTS = new Set([...GUIDE_TYPES_TO_HANDLE_BY_BRAND, ...Object.keys(GUIDE_GLOBAL_TYPES)])
+export const DELETED_USER = 'deleted user'
+
+type ValidAuditRes = {
+  // eslint-disable-next-line camelcase
+  audit_logs: { created_at: string; actor_name: string }[]
+}
+
+const AUDIT_SCHEMA = Joi.object({
+  audit_logs: Joi.array().items(Joi.object({
+    created_at: Joi.string().required(),
+    actor_name: Joi.string().required().not(''),
+  }).unknown(true)).required(),
+}).unknown(true).required()
+
+const isValidAuditRes = createSchemeGuard<ValidAuditRes>(
+  AUDIT_SCHEMA, 'Received an invalid value for audit_logs response'
+)
+
+const getLastAuditTime = async (client: ZendeskClient): Promise<string | undefined> => {
+  try {
+    const res = (await client.getSinglePage({
+      url: '/api/v2/audit_logs',
+      queryParams: {
+        'page[size]': '1',
+        // this is the log creation time and not when the source was created
+        sort: '-created_at',
+      },
+    })).data
+    if (isValidAuditRes(res)) {
+      return res.audit_logs[0].created_at
+    }
+  } catch (e) {
+    log.error(`could not get the last audit_log, getSinglePage returned an error'. error: ${e}`)
+  }
+  log.error('could not get the last audit_log, the result of getSinglePage was not valid.')
+  return undefined
+}
+
+const createTimeElements = async (lastAuditTime: string): Promise<Element[]> => {
+  const auditTimeType = new ObjectType({
+    elemID: AUDIT_TIME_TYPE_ID,
+    isSettings: true,
+    fields: {
+      time: { refType: BuiltinTypes.STRING },
+    },
+    path: [ZENDESK, elementsUtils.TYPES_PATH, AUDIT_TIME_TYPE_NAME],
+    annotations: {
+      [CORE_ANNOTATIONS.HIDDEN]: true,
+    },
+  })
+  const instance = new InstanceElement(
+    ElemID.CONFIG_NAME,
+    auditTimeType,
+    {
+      time: lastAuditTime,
+    },
+    undefined,
+    { [CORE_ANNOTATIONS.HIDDEN]: true },
+  )
+  return [auditTimeType, instance]
+}
+
+const getChangedByName = async ({
+  instance, client, start, end,
+}:{
+  instance: InstanceElement
+  client: ZendeskClient
+  start: string
+  end: string
+})
+  : Promise<string | undefined> => {
```

Comment:
tiny nit: can consolidate
```suggestion
}) : Promise<string | undefined> => {
```